aws/provider.go

```diff
@@ -474,6 +474,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_nat_gateway":                              resourceAwsNatGateway(),
 			"aws_network_acl":                              resourceAwsNetworkAcl(),
 			"aws_default_network_acl":                      resourceAwsDefaultNetworkAcl(),
+			"aws_neptune_parameter_group":                  resourceAwsNeptuneParameterGroup(),
```

Comment:
This new resource is missing documentation:
* A new documentation page: `website/docs/r/neptune_parameter_group.html.markdown`
* A sidebar link in `website/aws.erb`